tests/product/tests/test_kube_api_rules.py

```diff
@@ -10,31 +10,32 @@
 from product.tests.kube_rules import *
 
 
+@pytest.mark.k8s_object_rules
 @pytest.mark.parametrize(
     ("rule_tag", "resource_type", "resource_body", "expected"),
     [
-        *cis_5_1_3.values(),
-        *cis_5_1_5.values(),
-        *cis_5_1_6.values(),
-        *cis_5_2_2.values(),
-        *cis_5_2_3.values(),
-        *cis_5_2_4.values(),
-        *cis_5_2_5.values(),
-        *cis_5_2_6.values(),
+        # *cis_5_1_3.values(),
+        # *cis_5_1_5.values(),
+        # *cis_5_1_6.values(),
+        # *cis_5_2_2.values(),
+        # *cis_5_2_3.values(),
+        # *cis_5_2_4.values(),
+        # *cis_5_2_5.values(),
+        # *cis_5_2_6.values(),
         *cis_5_2_7.values(),
         # *cis_5_2_8.values(),
         # *cis_5_2_9.values(),
         # *cis_5_2_10.values(),
     ],
     ids=[
-        *cis_5_1_3.keys(),
-        *cis_5_1_5.keys(),
-        *cis_5_1_6.keys(),
-        *cis_5_2_2.keys(),
-        *cis_5_2_3.keys(),
-        *cis_5_2_4.keys(),
-        *cis_5_2_5.keys(),
-        *cis_5_2_6.keys(),
+        # *cis_5_1_3.keys(),
+        # *cis_5_1_5.keys(),
+        # *cis_5_1_6.keys(),
+        # *cis_5_2_2.keys(),
+        # *cis_5_2_3.keys(),
+        # *cis_5_2_4.keys(),
+        # *cis_5_2_5.keys(),
+        # *cis_5_2_6.keys(),
         *cis_5_2_7.keys(),
         # *cis_5_2_8.keys(),
         # *cis_5_2_9.keys(),
```

Comment:
Please add also brief explanation why test cases are commented

Reply:
That was just for debugging, updated now.